Dockerfile

```diff
@@ -1,4 +1,5 @@
 ARG CUDA_VERSION
+ARG XGB_HASH=6d293020fbfa2c67b532d550fe5d55689662caac
```

Comment:
We don't need to hard code this right?

Reply:
This is necessary so that we can explicitly invalidate the Docker layer and force recompile XGBoost.